src/main/java/com/minecolonies/coremod/colony/requestsystem/data/StandardRequestSystemBuildingDataStore.java

```diff
@@ -0,0 +1,212 @@
+                                                 .map(controller::serialize)
+                                                 .collect(NBTUtils.toNBTTagList()));
+
+                return entryCompound;
+            }).collect(NBTUtils.toNBTTagList()));
+            compound.setTag(TAG_OPEN_REQUESTS_BY_CITIZEN, standardRequestSystemBuildingDataStore.openRequestsByCitizen.keySet().stream().map(integer -> {
+                final NBTTagCompound entryCompound = new NBTTagCompound();
+
+                entryCompound.setTag(TAG_TOKEN, controller.serialize(integer));
+                entryCompound.setTag(TAG_LIST, standardRequestSystemBuildingDataStore.openRequestsByCitizen.get(integer)
+                                                 .stream()
+                                                 .map(controller::serialize)
+                                                 .collect(NBTUtils.toNBTTagList()));
+
+                return entryCompound;
+            }).collect(NBTUtils.toNBTTagList()));
+            compound.setTag(TAG_COMPLETED_REQUESTS_BY_CITIZEN, standardRequestSystemBuildingDataStore.completedRequestsByCitizen.keySet().stream().map(integer -> {
+                final NBTTagCompound entryCompound = new NBTTagCompound();
+
+                entryCompound.setTag(TAG_TOKEN, controller.serialize(integer));
+                entryCompound.setTag(TAG_LIST, standardRequestSystemBuildingDataStore.completedRequestsByCitizen.get(integer)
+                                                 .stream()
+                                                 .map(controller::serialize)
+                                                 .collect(NBTUtils.toNBTTagList()));
+
+                return entryCompound;
+            }).collect(NBTUtils.toNBTTagList()));
+            compound.setTag(TAG_CITIZEN_BY_OPEN_REQUEST, standardRequestSystemBuildingDataStore.citizenByOpenRequest.keySet().stream().map(iToken -> {
+                final NBTTagCompound entryCompound = new NBTTagCompound();
+
+                entryCompound.setTag(TAG_TOKEN, controller.serialize(iToken));
+                entryCompound.setTag(TAG_VALUE, controller.serialize(standardRequestSystemBuildingDataStore.citizenByOpenRequest.get(iToken)));
+
+                return entryCompound;
+            }).collect(NBTUtils.toNBTTagList()));
+
+            return compound;
+        }
+
+        @NotNull
+        @Override
+        public StandardRequestSystemBuildingDataStore deserialize(@NotNull final IFactoryController controller, @NotNull final NBTTagCompound nbt) throws Throwable
+        {
+            final IToken<?> token = controller.deserialize(nbt.getCompoundTag(TAG_TOKEN));
+            final Map<TypeToken<?>, Collection<IToken<?>>> openRequestsByRequestableType = NBTUtils
+                                                                                             .streamCompound(nbt.getTagList(TAG_OPEN_REQUESTS_BY_TYPE, Constants.NBT.TAG_COMPOUND)).map(nbtTagCompound -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 203 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)